android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlReceiver.java

```diff
@@ -4,16 +4,28 @@
 import android.content.Intent;
 import android.os.Build;
 
+import com.smartdevicelink.proxy.rpc.VehicleType;
```

Comment:
Please remove changes to this file as they relate to hello sdl

Reply:
@JulianKast Done. (Please, see c35b6f5)